unified-codes/apps/web/src/components/templates/Details.tsx

```diff
@@ -11,10 +11,14 @@ export interface DetailsProps {
  entity?: IEntity
 }
 
+interface IDetailsParameters {
+  code: string;
+}
+
 export type Details = React.FunctionComponent<DetailsProps>;
 
 export const DetailsComponent: Details = ({ entity, getEntity }) => {
-  const { code } = useParams();
+  const { code } = useParams<IDetailsParameters>();
```

Comment:
I guess this is a dependency of sorts 🤔 . Like the change though 👍

Reply:
soz! I should have confessed that offroad! typescript whined, so I tinkered.